src/booking-list/components/BookingDetails.tsx

```diff
@@ -0,0 +1,106 @@
+  const params = useParams();
+
+  const [trigger, result] = useLazyGetBookingsIdQuery();
+
+  const idParams = params.bookingId;
+
+  const [rideCompletionText, setRideCompletionText] = useState('');
+
+  useEffect(() => {
+    if (idParams) {
+      const parsedParams = idParams.split('$');
+      const username = parsedParams[0];
+      const carId = parsedParams[1];
+      const timeSec = parsedParams[2];
+
+      trigger({ username, carId, startTime: timeSec });
+
+      if (result.isSuccess) {
+        setRideCompletionText(
+          result.currentData.booking.bookingOwner.settings.rideCompletionText,
+        );
+      }
+    }
+  }, [idParams, result.currentData, result.isSuccess, trigger]);
+
+  return (
+    <div className={'bookingPage'}>
+      {result.isLoading ? (
+        <Loading />
+      ) : (
+        <>
+          <div className={'bookingWrapper'}>
+            {result.isSuccess && (
+              <table>
+                <caption className={'bookingHeader cellDecoration'}>
+                  Booking details
+                </caption>
+                <tbody>
+                  <tr>
+                    <td className={'cellDecoration'}>Инициатор поездки</td>
+                    <td className={'cellDecoration'}>
+                      {result.currentData.booking.bookingOwner.firstName}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>Автомобиль</td>
+                    <td className={'cellDecoration'}>
+                      {result.currentData.booking.bookingOwner.availableCars}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>Время начала поездки</td>
+                    <td className={'cellDecoration'}>
+                      {datesManager.getFormattingDateTime(
+                        +new Date(result.currentData.booking.bookingStartTime),
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>
+                      Время завершения поездки
+                    </td>
+                    <td className={'cellDecoration'}></td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>Описание поездки</td>
+                    <td className={'cellDecoration'}>
+                      {result.currentData.booking.bookingDescription}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>Поездка завершена?</td>
+                    <td className={'cellDecoration'}></td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>
+                      Местонахождение автомобиля по окончании поездки
+                    </td>
+                    <td className={'cellDecoration'}></td>
```

Comment:
We could hide this field if the ride is not finished

Reply:
hid